core/templates/pages/admin-page/navbar/admin-navbar.component.html

```diff
@@ -170,7 +169,8 @@
                          alt="">
                   </span>
                   <div *ngIf="!profilePictureDataUrl" class="dropdown-toggle">
-                    <i class="material-icons md-40 oppia-dropdown-toggle-icon">&#xE853;</i>
+                    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css" integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
I don't think you need to import the stylesheet from CDN, you can directly use the Font Awesome classes. Remove this import from here and everywhere else.